ccpage178.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Write a class called IceCreamStand that inherits from the Restaurant class.
+# Add an attribute called flavors that stores a list of ice cream flavors.
+# Write a method that displays these flavors.
+# Create an instance of IceCreamStand, and call this method.
+class Restaurant():
+    """An attempt to model a restaurant"""
+    def __init__(self, name, cuisine):
+        """Initialize the attributes"""
+        self.name = name
+        self.cuisine = cuisine
+    
+    def describe_restaurant(self):
+        """Write two sentences to describe restaurant"""
+        print(f"The restaurant is called {self.name.title()}.")
+        print(f"{self.name.title()} serves {self.cuisine.title()}.")
+    
+    def open(self):
+        """Indicate whether or not the restaurant is open"""
+        print(f"{self.name.title()} is open.")
+    
+class IceCreamStand(Restaurant):
+    """An attempt to model a specific type of restaurant, an ice cream shop"""
+    def __init__(self, name, cuisine):
+        """Initialize the attributes"""
+        super().__init__(name, cuisine)
+        self.flavors = ['chocolate', 'vanilla', 'strawberry']
+    
+    def ice_cream_flavors(self):
+        """List the ice cream flavors offered"""
+        flavors_offered = ['chocolate', 'vanilla', 'strawberry']
+        print("These are the flavors offered: ")
+        for flavor in flavors_offered:
+            print("-" + flavor.title())
+        
+
+my_icecream = IceCreamStand('whits', 'ice cream')
+print(my_icecream.ice_cream_flavors())
+
+
```

Comment:
@paulghaddad One issue that I'm running into consistently is this: when I run the program a line prints out "None" and I'm not sure why? In addition to the lines I want to print, the word "None" shows up and I can't figure out why?